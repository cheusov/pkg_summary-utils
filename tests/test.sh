#!/bin/sh

LC_ALL=C
export LC_ALL

#
srcdir=..
if test "$OBJDIR"; then
    objdir=${OBJDIR}
else
    objdir='.'
fi

#
if test -z "$BMAKE"; then
    BMAKE=/usr/bin/make
fi
PKGSRCDIR="`pwd`/../../../.."

AWKPATH="$srcdir"
PATH=$OBJDIR:$PATH

export PKGSRCDIR BMAKE AWKPATH PATH

#
print_args (){
#    echo "$@"
    for i in "$@"; do
	printf " '%s'" "$i"
    done
}

runtest (){
    echo '--------------------------------------------------'
    printf '%s' '------- args:'
    print_args "$@"
    printf '\n'

    prog="$objdir/$1"
    shift

    "$prog" "$@" 2>&1
}

# pkg_grep_summary
runtest pkg_grep_summary PKGNAME 'fvalue ~ /^d/' < src_summary.txt
runtest pkg_grep_summary COMMENT \
    'tolower(fvalue) ~ /dictionary.*client/' < src_summary.txt | \
    grep -E 'PKGNAME|---'
runtest pkg_grep_summary -e EXFIELD < src_summary.txt | \
    grep -E 'PKGNAME|---'

# pkg_cmp_summary
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.1'
pkg_cmp_summary src_summary.txt src_summary2.txt | sort -k2,2
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.2.1'
pkg_cmp_summary -p src_summary.txt src_summary2.txt | sort -k2,2
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.2.2'
pkg_cmp_summary --with-pkgpath src_summary.txt src_summary2.txt | sort -k2,2
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.3'
pkg_cmp_summary src_summary4.txt src_summary5.txt | sort -k2,2
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.4'
pkg_cmp_summary -p src_summary4.txt src_summary5.txt | sort -k2,2
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.5.1'
pkg_cmp_summary --use-checksum src_summary4.txt src_summary5.txt | sort -k2,2
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.5.2'
pkg_cmp_summary -c src_summary4.txt src_summary5.txt | sort -k2,2
echo '--------------------------------------------------'
echo '------- pkg_cmp_summary #1.6'
pkg_cmp_summary -cp src_summary4.txt src_summary6.txt | sort -k2,2

# pkg_list_all_pkgs
echo '--------------------------------------------------'
echo '------- pkg_list_all_pkgs #3'
env  pkg_list_all_pkgs |
awk 'END {gsub(/[0-9]/, "X", NR); print NR}'

normalize_version (){
  awk '{
   gsub(/(nb|alpha|beta|pre|rc|pl)[0-9]+/, "")
   gsub(/-[0-9]+([.][0-9]+)*/, "-X")
   gsub(/jpeg-X.*$/, "jpeg-X")
   print $0
  }' "$@"
}

# pkg_micro_src_summary
echo '--------------------------------------------------'
echo '------- pkg_micro_src_summary #4'
pkgs="`sed -n 's/^PKGPATH=//p' src_summary.txt`"
pkg_micro_src_summary $pkgs | tee "$objdir"/summary_micro.txt |
normalize_version

# pkg_refresh_summary
echo '--------------------------------------------------'
echo '------- pkg_refresh_summary #5'
pkg_refresh_summary src_summary.txt src_summary2.txt |
sed -n 's/^PKGNAME=//p' | sort

# pkg_src_summary
echo '--------------------------------------------------'
echo '------- pkg_src_summary #6'
pkgs="`sed -n 's/^PKGPATH=//p' src_summary.txt`"
pkg_src_summary -f PKGNAME,PKGPATH $pkgs |
tee "$objdir"/summary_full.txt |
normalize_version

echo '--------------------------------------------------'
echo '------- pkg_src_summary #7'
pkg_cmp_summary -p "$objdir"/summary_micro.txt "$objdir"/summary_full.txt |
grep -v '^='

# pkg_summary4view
echo '--------------------------------------------------'
echo '------- pkg_summary4view #8'
pkg_grep_summary PKGPATH 'fvalue == "wip/pkg_summary-utils"' \
    < src_summary.txt | pkg_summary4view

echo '--------------------------------------------------'
echo '------- pkg_summary4view #8.1'
pkg_summary4view -h     2>&1 | grep pkg_summary4view >/dev/null && echo ok
pkg_summary4view --help 2>&1 | grep pkg_summary4view >/dev/null && echo ok

# pkg_uniq_summary
echo '--------------------------------------------------'
echo '------- pkg_uniq_summary #9'
pkg_uniq_summary src_summary3.txt

echo '--------------------------------------------------'
echo '------- pkg_uniq_summary #9.1'
pkg_uniq_summary -h 2>&1     | grep pkg_uniq_summary > /dev/null && echo ok1
pkg_uniq_summary --help 2>&1 | grep pkg_uniq_summary > /dev/null && echo ok2

# pkg_src_summary
echo '--------------------------------------------------'
echo '------- pkg_src_summary #10.1'
pkg_src_summary -m -fPKGNAME,PKGPATH www/ap2-python |
grep -v DEPENDS

echo '--------------------------------------------------'
echo '------- pkg_src_summary #10.2'
pkg_src_summary -m --fields PKGNAME,PKGPATH www/ap2-python:PKG_APACHE=apache2 |
grep -v DEPENDS

echo '--------------------------------------------------'
echo '------- pkg_src_summary #10.3'
pkg_src_summary -m --fields='PKGNAME PKGPATH' www/ap2-python:PYTHON_VERSION_REQD=25 |
grep -v DEPENDS

echo '--------------------------------------------------'
echo '------- pkg_src_summary #10.4'
pkg_src_summary -m -f'PKGNAME PKGPATH' \
   www/ap2-python:PYTHON_VERSION_REQD=25,PKG_APACHE=apache22 |
grep -v DEPENDS

echo '--------------------------------------------------'
echo '------- pkg_src_summary #11'
pkg_src_summary -A -f PKGNAME,PKGPATH \
   graphics/py-cairo:PYTHON_VERSION_REQD=25 |
grep -v DEPENDS |
pkg_grep_summary PKGPATH 'fvalue ~ /gmake|py-Numeric|py-cairo/' |
normalize_version

echo '--------------------------------------------------'
echo '------- pkg_assignments2pkgpath #12'
pkg_assignments2pkgpath -h 2>&1 |
grep pkg_assignments2pkgpath > /dev/null && echo ok1

pkg_assignments2pkgpath --help 2>&1 |
grep pkg_assignments2pkgpath > /dev/null && echo ok2

echo '--------------------------------------------------'
echo '------- pkg_src_fetch_var #13.1'
echo x11/xxkb | pkg_src_fetch_var -f 'PKGNAME PKGPATH MAINTAINER' |
normalize_version

echo '--------------------------------------------------'
echo '------- pkg_src_fetch_var #13.2'
echo x11/xxkb | pkg_src_fetch_var -v'PKGNAME PKGPATH MAINTAINER' |
normalize_version

echo '--------------------------------------------------'
echo '------- pkg_src_fetch_var #13.3'
echo x11/xxkb | pkg_src_fetch_var --vars='PKGNAME PKGPATH MAINTAINER' |
normalize_version

echo '--------------------------------------------------'
echo '------- pkg_src_fetch_var #13.4'
echo x11/xxkb | pkg_src_fetch_var --fields 'PKGNAME PKGPATH MAINTAINER' |
normalize_version

echo '--------------------------------------------------'
echo '------- pkg_micro_src_summary #14.1'
pkg_micro_src_summary -f 'PKGNAME PKGPATH MAINTAINER' x11/xxkb |
normalize_version
echo '--------------------------------------------------'
echo '------- pkg_micro_src_summary #14.2'
pkg_micro_src_summary --fields='PKGNAME PKGPATH MAINTAINER' x11/xxkb |
normalize_version
echo '--------------------------------------------------'
echo '------- pkg_micro_src_summary #14.3'
pkg_micro_src_summary -f'PKGNAME PKGPATH MAINTAINER' x11/xxkb |
normalize_version
echo '--------------------------------------------------'
echo '------- pkg_micro_src_summary #14.4'
pkg_micro_src_summary --fields 'PKGNAME PKGPATH MAINTAINER' x11/xxkb |
normalize_version
