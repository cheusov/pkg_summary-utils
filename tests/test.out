--------------------------------------------------
------- args: 'pkg_grep_summary' 'PKGNAME' 'fvalue ~ /^d/'
PKGNAME=distbb-0.22.0
PKGPATH=wip/distbb
DEPENDS=pkg_summary-utils>=0.18.1:../../wip/pkg_summary-utils paexec>=0.10.0:../../wip/paexec runawk>=0.13.0:../../wip/runawk
BUILD_DEPENDS= checkperms>=1.1:../../sysutils/checkperms
HOMEPAGE=http://mova.org/~cheusov/pub/distbb/
COMMENT=DISTributed Bulk Build tool for pkgsrc
MAINTAINER=cheusov@tut.by
CATEGORIES=pkgtools
EXFIELD=XVALUE1

PKGPATH=textproc/dictem
DEPENDS=dict-client>=1.9.14:../../textproc/dict-client emacs>=22:../../editors/emacs  emacs>=22:../../editors/emacs
BUILD_DEPENDS= checkperms>=1.1:../../sysutils/checkperms
CONFLICTS=xemacs-dictem-[0-9]*
HOMEPAGE=http://freshmeat.net/projects/dictem/
COMMENT=Dictionary client (RFC-2229) for [X]Emacs
MAINTAINER=cheusov@tut.by
CATEGORIES=textproc editors
PKGNAME=dictem-0.82

BUILD_DEPENDS= libtool-base>=1.5.18nb5:../../devel/libtool-base gmake>=3.81:../../devel/gmake checkperms>=1.1:../../sysutils/checkperms
PKGNAME=dict-server-1.10.11nb2
PKGPATH=wip/dict-server
DEPENDS=   libmaa>=0.990:../../devel/libmaa
HOMEPAGE=http://www.dict.org/
COMMENT=Dictionary Service Protocol server
MAINTAINER=cheusov@tut.by
CATEGORIES=textproc

PKGNAME=dict-client-1.9.15nb2
PKGPATH=textproc/dict-client
DEPENDS=  libltdl>=1.5.10:../../devel/libltdl
BUILD_DEPENDS= gmake>=3.81:../../devel/gmake checkperms>=1.1:../../sysutils/checkperms
HOMEPAGE=http://www.dict.org/
COMMENT=Dictionary Service Protocol client
MAINTAINER=pkgsrc-users@NetBSD.org
CATEGORIES=textproc

PKGNAME=dict-client-1.10.11nb2
PKGPATH=wip/dict-client
DEPENDS=  libmaa>=0.990:../../devel/libmaa
BUILD_DEPENDS= libtool-base>=1.5.18nb5:../../devel/libtool-base gmake>=3.81:../../devel/gmake checkperms>=1.1:../../sysutils/checkperms
HOMEPAGE=http://www.dict.org/
COMMENT=Dictionary Service Protocol client
MAINTAINER=cheusov@tut.by
CATEGORIES=textproc

--------------------------------------------------
------- args: 'pkg_grep_summary' 'COMMENT' 'tolower(fvalue) ~ /dictionary.*client/'
PKGNAME=dictem-0.82
PKGNAME=dict-client-1.9.15nb2
PKGNAME=dict-client-1.10.11nb2
--------------------------------------------------
------- args: 'pkg_grep_summary' '-e' 'EXFIELD'
PKGNAME=dictem-0.82
PKGNAME=pkg_online-0.5.0nb2
PKGNAME=dict-server-1.10.11nb2
PKGNAME=pkg_online-server-0.5.0
PKGNAME=dict-client-1.9.15nb2
PKGNAME=pkg_online-client-0.5.0
PKGNAME=libmaa-1.0.1nb1
PKGNAME=gmake-3.81
PKGNAME=paexec-0.10.0nb1
PKGNAME=libtool-base-1.5.24nb6
PKGNAME=runawk-0.14.3
PKGNAME=emacs-22.1nb6
PKGNAME=pkg_summary-utils-0.18.1
PKGNAME=libungif-4.1.4nb1
PKGNAME=tiff-3.8.2nb4
PKGNAME=x11-links-0.38
PKGNAME=perl-5.8.8nb8
PKGNAME=libltdl-1.5.24
PKGNAME=dict-client-1.10.11nb2
PKGNAME=pipestatus-0.4.0
PKGNAME=awk-pkgsrc-dewey-0.5.6
PKGNAME=png-1.2.32beta01
PKGNAME=netcat-1.10nb2
PKGNAME=pkg-config-0.23
PKGNAME=jpeg-6bnb4
------- args: 'pkg_grep_summary' '-m' 'MAINTAINER' 'cheusov|vle@gmx.net'
PKGNAME=distbb-0.22.0
PKGPATH=wip/distbb
COMMENT=DISTributed Bulk Build tool for pkgsrc
MAINTAINER=cheusov@tut.by

PKGPATH=textproc/dictem
COMMENT=Dictionary client (RFC-2229) for [X]Emacs
MAINTAINER=cheusov@tut.by
PKGNAME=dictem-0.82

COMMENT=Client/Server search in pkgsrc packages
MAINTAINER=cheusov@tut.by
PKGNAME=pkg_online-0.5.0nb2
PKGPATH=wip/pkg_online

PKGNAME=dict-server-1.10.11nb2
PKGPATH=wip/dict-server
COMMENT=Dictionary Service Protocol server
MAINTAINER=cheusov@tut.by

PKGNAME=pkg_online-server-0.5.0
PKGPATH=wip/pkg_online-server
COMMENT=Client/Server search in pkgsrc packages (server tools)
MAINTAINER=cheusov@tut.by

PKGNAME=pkg_online-client-0.5.0
PKGPATH=wip/pkg_online-client
COMMENT=Client/Server search in pkgsrc packages (client tools)
MAINTAINER=cheusov@tut.by

PKGNAME=libmaa-1.0.1nb1
PKGPATH=devel/libmaa
COMMENT=Provides a few data structures and helpful functions
MAINTAINER=cheusov@tut.by

PKGNAME=paexec-0.10.0nb1
PKGPATH=wip/paexec
COMMENT=Parallel executor, distributes tasks over network
MAINTAINER=cheusov@tut.by

PKGNAME=runawk-0.14.3
PKGPATH=wip/runawk
COMMENT=Wrapper that impelements modules for AWK
MAINTAINER=cheusov@tut.by

PKGNAME=pkg_summary-utils-0.18.1
PKGPATH=wip/pkg_summary-utils
COMMENT=Utilities for manipulating pkg_summary(5) files
MAINTAINER=cheusov@tut.by

PKGNAME=dict-client-1.10.11nb2
PKGPATH=wip/dict-client
COMMENT=Dictionary Service Protocol client
MAINTAINER=cheusov@tut.by

PKGNAME=pipestatus-0.4.0
PKGPATH=wip/pipestatus
COMMENT=UNIX/POSIX shell helper for running pipes safely
MAINTAINER=cheusov@tut.by

PKGNAME=awk-pkgsrc-dewey-0.5.6
PKGPATH=wip/awk-pkgsrc-dewey
COMMENT=Pkgsrc dewey module for AWK programming language
MAINTAINER=cheusov@tut.by

--------------------------------------------------
------- args: 'pkg_grep_summary' '-s' 'PKGPATH' 'graphics/png'
PKGNAME=png-1.2.32beta01
PKGPATH=graphics/png
BUILD_DEPENDS= libtool-base>=1.5.18nb5:../../devel/libtool-base checkperms>=1.1:../../sysutils/checkperms
HOMEPAGE=http://www.libpng.org/pub/png/
COMMENT=Library for manipulating PNG images
MAINTAINER=wiz@NetBSD.org
CATEGORIES=graphics

--------------------------------------------------
------- pkg_cmp_summary #1.1
= awk-pkgsrc-dewey 0.5.6 0.5.6
= checkperms 1.10 1.10
2 dict-client
< dict-server 1.10.11nb2 1.11.0
= dictem 0.82 0.82
+ digest 99.99.99
= distbb 0.22.0 0.22.0
= emacs 22.1nb6 22.1nb6
= gmake 3.81 3.81
= jpeg 6bnb4 6bnb4
= libltdl 1.5.24 1.5.24
= libmaa 1.0.1nb1 1.0.1nb1
= libtool-base 1.5.24nb6 1.5.24nb6
= libungif 4.1.4nb1 4.1.4nb1
- netcat 1.10nb2
= paexec 0.10.0nb1 0.10.0nb1
= perl 5.8.8nb8 5.8.8nb8
< pipestatus 0.4.0 0.5.0
= pkg-config 0.23 0.23
= pkg_online 0.5.0nb2 0.5.0nb2
= pkg_online-client 0.5.0 0.5.0
= pkg_online-server 0.5.0 0.5.0
= pkg_summary-utils 0.18.1 0.18.1
= png 1.2.32beta01 1.2.32beta01
> runawk 0.14.3 0.13.0
= tiff 3.8.2nb4 3.8.2nb4
- x11-links 0.38
--------------------------------------------------
------- pkg_cmp_summary #1.2.1
= devel/gmake gmake 3.81 3.81
= devel/libltdl libltdl 1.5.24 1.5.24
= devel/libmaa libmaa 1.0.1nb1 1.0.1nb1
= devel/libtool-base libtool-base 1.5.24nb6 1.5.24nb6
= devel/pkg-config pkg-config 0.23 0.23
= editors/emacs emacs 22.1nb6 22.1nb6
= graphics/jpeg jpeg 6bnb4 6bnb4
= graphics/libungif libungif 4.1.4nb1 4.1.4nb1
= graphics/png png 1.2.32beta01 1.2.32beta01
= graphics/tiff tiff 3.8.2nb4 3.8.2nb4
= lang/perl5 perl 5.8.8nb8 5.8.8nb8
- net/netcat netcat 1.10nb2
+ pkgtools/digest digest 99.99.99
- pkgtools/x11-links x11-links 0.38
= sysutils/checkperms checkperms 1.10 1.10
= textproc/dict-client dict-client 1.9.15nb2 1.9.15nb2
= textproc/dictem dictem 0.82 0.82
= wip/awk-pkgsrc-dewey awk-pkgsrc-dewey 0.5.6 0.5.6
= wip/dict-client dict-client 1.10.11nb2 1.10.11nb2
< wip/dict-server dict-server 1.10.11nb2 1.11.0
= wip/distbb distbb 0.22.0 0.22.0
= wip/paexec paexec 0.10.0nb1 0.10.0nb1
< wip/pipestatus pipestatus 0.4.0 0.5.0
= wip/pkg_online pkg_online 0.5.0nb2 0.5.0nb2
= wip/pkg_online-client pkg_online-client 0.5.0 0.5.0
= wip/pkg_online-server pkg_online-server 0.5.0 0.5.0
= wip/pkg_summary-utils pkg_summary-utils 0.18.1 0.18.1
> wip/runawk runawk 0.14.3 0.13.0
--------------------------------------------------
------- pkg_cmp_summary #1.2.2
= devel/gmake gmake 3.81 3.81
= devel/libltdl libltdl 1.5.24 1.5.24
= devel/libmaa libmaa 1.0.1nb1 1.0.1nb1
= devel/libtool-base libtool-base 1.5.24nb6 1.5.24nb6
= devel/pkg-config pkg-config 0.23 0.23
= editors/emacs emacs 22.1nb6 22.1nb6
= graphics/jpeg jpeg 6bnb4 6bnb4
= graphics/libungif libungif 4.1.4nb1 4.1.4nb1
= graphics/png png 1.2.32beta01 1.2.32beta01
= graphics/tiff tiff 3.8.2nb4 3.8.2nb4
= lang/perl5 perl 5.8.8nb8 5.8.8nb8
- net/netcat netcat 1.10nb2
+ pkgtools/digest digest 99.99.99
- pkgtools/x11-links x11-links 0.38
= sysutils/checkperms checkperms 1.10 1.10
= textproc/dict-client dict-client 1.9.15nb2 1.9.15nb2
= textproc/dictem dictem 0.82 0.82
= wip/awk-pkgsrc-dewey awk-pkgsrc-dewey 0.5.6 0.5.6
= wip/dict-client dict-client 1.10.11nb2 1.10.11nb2
< wip/dict-server dict-server 1.10.11nb2 1.11.0
= wip/distbb distbb 0.22.0 0.22.0
= wip/paexec paexec 0.10.0nb1 0.10.0nb1
< wip/pipestatus pipestatus 0.4.0 0.5.0
= wip/pkg_online pkg_online 0.5.0nb2 0.5.0nb2
= wip/pkg_online-client pkg_online-client 0.5.0 0.5.0
= wip/pkg_online-server pkg_online-server 0.5.0 0.5.0
= wip/pkg_summary-utils pkg_summary-utils 0.18.1 0.18.1
> wip/runawk runawk 0.14.3 0.13.0
--------------------------------------------------
------- pkg_cmp_summary #1.3
= dictem 1.0.0 1.0.0
--------------------------------------------------
------- pkg_cmp_summary #1.4
- textproc/dictem dictem 1.0.0
+ wip/dictem dictem 1.0.0
--------------------------------------------------
------- pkg_cmp_summary #1.5.1
! dictem 1.0.0 1.0.0
--------------------------------------------------
------- pkg_cmp_summary #1.5.2
! dictem 1.0.0 1.0.0
--------------------------------------------------
------- pkg_cmp_summary #1.6
! textproc/dictem dictem 1.0.0 1.0.0
--------------------------------------------------
------- pkg_cmp_summary #1.7
= devel/gmake
= devel/libltdl
= devel/libmaa
= devel/libtool-base
= devel/pkg-config
= editors/emacs
= graphics/jpeg
= graphics/libungif
= graphics/png
= graphics/tiff
= lang/perl5
- net/netcat
+ pkgtools/digest
- pkgtools/x11-links
= sysutils/checkperms
= textproc/dict-client
= textproc/dictem
= wip/awk-pkgsrc-dewey
= wip/dict-client
= wip/dict-server
= wip/distbb
= wip/paexec
= wip/pipestatus
= wip/pkg_online
= wip/pkg_online-client
= wip/pkg_online-server
= wip/pkg_summary-utils
= wip/runawk
--------------------------------------------------
------- pkg_list_all_pkgs #3
XXXX
--------------------------------------------------
------- pkg_micro_src_summary #4
PKGNAME=distbb-X
PKGPATH=wip/distbb

PKGNAME=pkg_online-X
PKGPATH=wip/pkg_online

PKGNAME=dict-server-X
PKGPATH=wip/dict-server

PKGNAME=checkperms-X
PKGPATH=sysutils/checkperms

PKGNAME=dict-client-X
PKGPATH=textproc/dict-client

PKGNAME=libmaa-X
PKGPATH=devel/libmaa

PKGNAME=gmake-X
PKGPATH=devel/gmake

PKGNAME=paexec-X
PKGPATH=wip/paexec

PKGNAME=libtool-base-X
PKGPATH=devel/libtool-base

PKGNAME=runawk-X
PKGPATH=wip/runawk

PKGNAME=emacs-X
PKGPATH=editors/emacs

PKGNAME=pkg_summary-utils-X
PKGPATH=wip/pkg_summary-utils

PKGNAME=libungif-X
PKGPATH=graphics/libungif

PKGNAME=tiff-X
PKGPATH=graphics/tiff

PKGNAME=x11-links-X
PKGPATH=pkgtools/x11-links

PKGNAME=perl-X
PKGPATH=lang/perl5

PKGNAME=libltdl-X
PKGPATH=devel/libltdl

PKGNAME=dict-client-X
PKGPATH=wip/dict-client

PKGNAME=pipestatus-X
PKGPATH=wip/pipestatus

PKGNAME=awk-pkgsrc-dewey-X
PKGPATH=wip/awk-pkgsrc-dewey

PKGNAME=png-X
PKGPATH=graphics/png

PKGNAME=netcat-X
PKGPATH=net/netcat

PKGNAME=pkg-config-X
PKGPATH=devel/pkg-config

PKGNAME=jpeg-X
PKGPATH=graphics/jpeg

PKGNAME=dictem-X
PKGPATH=textproc/dictem

PKGNAME=pkg_online-server-X
PKGPATH=wip/pkg_online-server

PKGNAME=pkg_online-client-X
PKGPATH=wip/pkg_online-client

--------------------------------------------------
------- pkg_refresh_summary #5.1
awk-pkgsrc-dewey-0.5.6
checkperms-1.10
dict-client-1.10.11nb2
dict-client-1.9.15nb2
dict-server-1.11.0
dictem-0.82
digest-99.99.99
distbb-0.22.0
emacs-22.1nb6
gmake-3.81
jpeg-6bnb4
libltdl-1.5.24
libmaa-1.0.1nb1
libtool-base-1.5.24nb6
libungif-4.1.4nb1
netcat-1.10nb2
paexec-0.10.0nb1
perl-5.8.8nb8
pipestatus-0.5.0
pkg-config-0.23
pkg_online-0.5.0nb2
pkg_online-client-0.5.0
pkg_online-server-0.5.0
pkg_summary-utils-0.18.1
png-1.2.32beta01
runawk-0.14.3
tiff-3.8.2nb4
x11-links-0.38
--------------------------------------------------
------- pkg_refresh_summary #5.2
dict-server-1.10.11nb2
pipestatus-0.4.0
runawk-0.13.0
--------------------------------------------------
------- pkg_src_summary #6
PKGNAME=distbb-X
PKGPATH=wip/distbb

PKGNAME=dictem-X
PKGPATH=textproc/dictem

PKGNAME=pkg_online-X
PKGPATH=wip/pkg_online

PKGNAME=dict-server-X
PKGPATH=wip/dict-server

PKGNAME=checkperms-X
PKGPATH=sysutils/checkperms

PKGNAME=pkg_online-server-X
PKGPATH=wip/pkg_online-server

PKGNAME=dict-client-X
PKGPATH=textproc/dict-client

PKGNAME=pkg_online-client-X
PKGPATH=wip/pkg_online-client

PKGNAME=libmaa-X
PKGPATH=devel/libmaa

PKGNAME=gmake-X
PKGPATH=devel/gmake

PKGNAME=paexec-X
PKGPATH=wip/paexec

PKGNAME=libtool-base-X
PKGPATH=devel/libtool-base

PKGNAME=runawk-X
PKGPATH=wip/runawk

PKGNAME=emacs-X
PKGPATH=editors/emacs

PKGNAME=pkg_summary-utils-X
PKGPATH=wip/pkg_summary-utils

PKGNAME=libungif-X
PKGPATH=graphics/libungif

PKGNAME=tiff-X
PKGPATH=graphics/tiff

PKGNAME=x11-links-X
PKGPATH=pkgtools/x11-links

PKGNAME=perl-X
PKGPATH=lang/perl5

PKGNAME=libltdl-X
PKGPATH=devel/libltdl

PKGNAME=dict-client-X
PKGPATH=wip/dict-client

PKGNAME=pipestatus-X
PKGPATH=wip/pipestatus

PKGNAME=awk-pkgsrc-dewey-X
PKGPATH=wip/awk-pkgsrc-dewey

PKGNAME=png-X
PKGPATH=graphics/png

PKGNAME=netcat-X
PKGPATH=net/netcat

PKGNAME=pkg-config-X
PKGPATH=devel/pkg-config

PKGNAME=jpeg-X
PKGPATH=graphics/jpeg

--------------------------------------------------
------- pkg_src_summary #7
--------------------------------------------------
------- pkg_summary4view #8
-----------------------------------------------------------
PKGNAME:        pkg_summary-utils-0.18.1
PKGPATH:        wip/pkg_summary-utils
DEPENDS:        runawk>=0.12.1:../../wip/runawk
                awk-pkgsrc-dewey>=0.5.6:../../wip/awk-pkgsrc-dewey
                pipestatus-[0-9]*:../../wip/pipestatus
BUILD_DEPENDS:  checkperms>=1.1:../../sysutils/checkperms
CONFLICTS:      paexec<0.9.0
                pkg_src_summary-[0-9]*
                pkg_cmp_summary-[0-9]*
                pkg_src_update_summary-[0-9]*
                pkg_update_summary-[0-9]*
                pkg_summary4view-[0-9]*
HOMEPAGE:       http://www.mova.org/~cheusov/pub/pkg_summary-utils/
COMMENT:        Utilities for manipulating pkg_summary(5) files
MAINTAINER:     cheusov@tut.by
CATEGORIES:     pkgtools

--------------------------------------------------
------- pkg_summary4view #8.1
ok
ok
--------------------------------------------------
------- pkg_uniq_summary #9
ASSIGNMENTS=OPT1=1,OPT2=2
PKGNAME=distbb-0.22.0
PKGPATH=wip/distbb

PKGNAME=distbb-0.22.0
PKGPATH=wip/distbb:OPT=1,OPT2=3

--------------------------------------------------
------- pkg_uniq_summary #9.1
ok1
ok2
--------------------------------------------------
------- pkg_src_summary #10.1
ASSIGNMENTS=PKG_APACHE=apache2,PYTHON_VERSION_REQD=26
PKGNAME=ap2-py26-python-3.3.1
PKGPATH=www/ap2-python

ASSIGNMENTS=PKG_APACHE=apache2
PKGNAME=ap2-py25-python-3.3.1
PKGPATH=www/ap2-python

ASSIGNMENTS=PKG_APACHE=apache2,PYTHON_VERSION_REQD=24
PKGNAME=ap2-py24-python-3.3.1
PKGPATH=www/ap2-python

ASSIGNMENTS=PYTHON_VERSION_REQD=26
PKGNAME=ap22-py26-python-3.3.1
PKGPATH=www/ap2-python

PKGNAME=ap22-py25-python-3.3.1
PKGPATH=www/ap2-python

ASSIGNMENTS=PYTHON_VERSION_REQD=24
PKGNAME=ap22-py24-python-3.3.1
PKGPATH=www/ap2-python

--------------------------------------------------
------- pkg_src_summary #10.2
ASSIGNMENTS=PYTHON_VERSION_REQD=26
PKGNAME=ap22-py26-python-3.3.1
PKGPATH=www/ap2-python

PKGNAME=ap22-py25-python-3.3.1
PKGPATH=www/ap2-python

ASSIGNMENTS=PYTHON_VERSION_REQD=24
PKGNAME=ap22-py24-python-3.3.1
PKGPATH=www/ap2-python

--------------------------------------------------
------- pkg_src_summary #10.3
ASSIGNMENTS=PKG_APACHE=apache2
PKGNAME=ap2-py25-python-3.3.1
PKGPATH=www/ap2-python

PKGNAME=ap22-py25-python-3.3.1
PKGPATH=www/ap2-python

--------------------------------------------------
------- pkg_src_summary #10.4
PKGNAME=ap22-py25-python-3.3.1
PKGPATH=www/ap2-python

--------------------------------------------------
------- pkg_src_summary #11
PKGNAME=py25-cairo-X
PKGPATH=graphics/py-cairo

PKGNAME=py25-Numeric-X
PKGPATH=math/py-Numeric

--------------------------------------------------
------- pkg_assignments2pkgpath #12
ok1
ok2
--------------------------------------------------
------- pkg_src_fetch_var #13.1
+	xxkb-X	x11/xxkb	vle@gmx.net
--------------------------------------------------
------- pkg_src_fetch_var #13.2
+	xxkb-X	x11/xxkb	vle@gmx.net
--------------------------------------------------
------- pkg_src_fetch_var #13.3
+	xxkb-X	x11/xxkb	vle@gmx.net
--------------------------------------------------
------- pkg_src_fetch_var #13.4
+	xxkb-X	x11/xxkb	vle@gmx.net
--------------------------------------------------
------- pkg_micro_src_summary #14.1
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_micro_src_summary #14.2
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_micro_src_summary #14.3
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_micro_src_summary #14.4
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_src_summary #15.1
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_src_summary #15.2
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_src_summary #15.3
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_src_summary #15.4
PKGNAME=xxkb-X
PKGPATH=x11/xxkb
MAINTAINER=vle@gmx.net

--------------------------------------------------
------- pkg_src_summary #16.1
PKGNAME=xxkb-X
COMMENT=XXKB - switches and indicates a current keyboard layout

--------------------------------------------------
------- pkg_src_summary #16.2
PKGNAME=xxkb-X
COMMENT=XXKB - switches and indicates a current keyboard layout

--------------------------------------------------
------- pkg_src_summary #16.3
PKGNAME=xxkb-X
COMMENT=XXKB - switches and indicates a current keyboard layout

--------------------------------------------------
------- pkg_src_summary #16.4
PKGNAME=xxkb-X
COMMENT=XXKB - switches and indicates a current keyboard layout

